lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, figure):
         if DEBUG:
             print('FigureCanvasQtAgg: ', figure)
         FigureCanvasQT.__init__(self, figure)
+        FigureCanvasQTAggBase.__init__(self, figure)
```

Comment:
I was also a bit confused concerning that change from qt4 to qt5. They prefer cooperative multiple inheritance now, while I got used to explicitly/manually calling all init functions, which is required when using PyQt4 I guess.